app/src/main/java/de/westnordost/streetcomplete/data/user/achievements/AchievementsModule.kt

```diff
@@ -178,6 +178,14 @@ private val links = listOf(
         R.drawable.ic_link_organic_maps,
         R.string.link_organic_maps_description
     ),
+    Link(
+        "osmand",
+        "https://osmand.net/",
+        "OsmAnd",
+        LinkCategory.MAPS,
+        R.drawable.ic_link_osmand,
+        R.string.link_osmand_description
+    ),
```

Comment:
Is OsmAnd really completely open source? https://play.google.com/store/apps/details?id=net.osmand.plus costs something.

Reply:
Yes, that is why F-droid has version with all functionality available for free (though if Osmand operators would want this they could limit server access and require people to host own map servers).

Reply:
ah